Jenkinsfile

```diff
@@ -11,14 +11,35 @@ pipeline {
         dockerfile true
     }
 
+    parameters {
+        string(name: "REMOTE_SWF_LINK", defaultValue: "https://github.com/Pelagicore/software-factory.git")
+        //string(name: "REMOTE_SWF_BP_LINK", defaultValue: "https://github.com/Pelagicore/software-factory-blueprint.git")
+        string(name: "BRANCH_SWF", defaultValue: "master")
+    }
+
     stages {
         stage('Download') {
             steps {
+
                 // Delete old files
+
                 sh 'rm -rf .[^.] .??* *'
 
-                // Checkout the git repository and refspec pointed to by jenkins
-                checkout scm
+                // Get The latest SWF src
+
+                sh "git init || git init" // some times it fails to initialize so trying another time, in case it fails.
+                sh "git remote add jenkins ${REMOTE_SWF_LINK}"
```

Comment:
To properly show that the SWF and the Blueprint are built separately, I would prefer if these steps were also inside a `dir() { }` block.